src/components/CustomInputs/MultiLineInput.vue

```diff
@@ -41,17 +43,46 @@ export default {
     // Allows swapping between json input and yaml inputs
     mode(val) {
       if (val == 'json') {
-        this.jsonInput = jsBeautify(this.jsonInput, {
-          indent_size: 4,
-          space_in_empty_paren: true,
-          preserve_newlines: false
-        })
+        this.jsonInput = jsBeautify(
+          JSON.stringify(parse(this.yamlInput) || {}),
+          {
+            indent_size: 4,
+            space_in_empty_paren: true,
+            preserve_newlines: false
+          }
+        )
 
         // Use next tick to make sure the json input element exists
         this.$nextTick(() => {
           this.$refs['json-input'].validateJson()
         })
+      } else {
+        if (this.jsonInput === '{}') {
+          this.yamlInput = ''
+        } else {
+          this.yamlInput = stringify(JSON.parse(this.jsonInput))
```

Comment:
Not a major issue but do we need a try/catch here?  If I add yaml to the json input by mistake and try to switch to yaml, the input vanishes. 
Screen Recording 2021-05-18 at 4.53.10 PM

Reply:
https://user-images.githubusercontent.com/40272060/118722313-03e55400-b7fa-11eb-876a-ebd9dd782167.mov

Reply:
Hm yeah I think a try catch makes sense here - will add that

Reply:
Updated 👍🏻